source/server-discovery-and-monitoring/tests/integration/connectTimeoutMS.json

```diff
@@ -42,7 +42,8 @@
               },
               "data": {
                 "failCommands": [
-                  "isMaster"
+                  "isMaster",
+                  "hello"
```

Comment:
I see we're updating these tests but haven't updated the SDAM spec itself yet. Is that going to happen in a different ticket?

Reply:
I've added a change to cover the `isWritablePrimary` field. The SDAM spec will also see changes related to this change (in a separate DRIVERS ticket) to provide better error reporting in case the handshake command completes but yields an unexpected result (e.g. "hello" command not found when using Versioned API).